src/messaging/Message.ts

```diff
@@ -247,28 +262,46 @@ export interface IRequestAttestationForClaim extends IMessageBodyBase {
   type: MessageBodyType.REQUEST_ATTESTATION_FOR_CLAIM
 }
 export interface ISubmitAttestationForClaim extends IMessageBodyBase {
-  content: IAttestedClaim
+  content: {
+    attestation: IAttestation
+    attestationPE?: AttestationPE
+  }
   type: MessageBodyType.SUBMIT_ATTESTATION_FOR_CLAIM
 }
 export interface IRejectAttestationForClaim extends IMessageBodyBase {
-  content: IRequestForAttestation
+  content: false
```

Comment:
Since it is async, we need to find a way to identify the message.
Either `inReplyTo` has to be required, or we use session ids, from the actors

Reply:
Can also be a ticket

Reply:
tracked here KILTprotocol/ticket/issues/519